SerialPrograms/Source/Integrations/DiscordIntegrationSettings.cpp

```diff
@@ -204,16 +204,16 @@ DiscordIntegrationSettingsWidget::DiscordIntegrationSettingsWidget(QWidget& pare
         button_start, &QPushButton::clicked,
         this, [&value](bool) {
             switch (value.library0){
-#ifdef PA_SLEEPY
             case DiscordIntegrationSettingsOption::Library::SleepyDiscord:
+#ifdef PA_SLEEPY

```

Comment:
Why are you moving these if-defs? These Sleepy cases are not supposed to be compiled when it's disabled.

Reply:
because otherwise compilation failed due to `-Werror`

```
[  1%] Building CXX object CMakeFiles/SerialPrograms.dir/Source/Integrations/DiscordIntegrationSettings.cpp.o
build/Arduino-Source/SerialPrograms/Source/Integrations/DiscordIntegrationSettings.cpp:125:13: error: enumeration value 'SleepyDiscord' not handled in switch [-Werror,-Wswitch]
    switch (library0){
            ^~~~~~~~
build/Arduino-Source/SerialPrograms/Source/Integrations/DiscordIntegrationSettings.cpp:125:13: note: add missing switch cases
    switch (library0){
            ^
1 error generated.
```

Reply:
You can use a `default;` to swallow everything else.